ipmof_energymap.py

```diff
@@ -55,5 +47,18 @@
     print('-----------------------------------------------------------------------------------')
     print(mof_index, 'Calculating energy map for ->', mof.name)
 
-    # Calculate energy map -------------------------------------------------------------------------
-    job_queue.enqueue(energy_map, sim_par, mof, atom_list)
+    # Submit jobs here
+    if sys.argv[-1] == 'q':
+        # Load job server libraries
+        from rq import Queue
+        from redis import Redis
+        import sjs
+
```

Comment:
not PEP8, but not sure if we really care enough here since these are in the command scripts, not the library.